### PR TITLE
Trading equipment - Stock endpoints implemented 

### PR DESCRIPTION
**Pull request by ikamaci**
Error handling has not implemented yet but in order to start Portfolio part this pr can be used

### CODE REVIEW

File: backend/models/stock.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        type: String,
+        required:true
+    },
+    price:{
+        type: Number,
+        required: true
+    },
+
+    monthlyPrice:{
+        type: {}
```
Comment (mdumandag) [inline backend/models/stock.js]: Is this  an array ? If so, why dont we specify it as `[Number]`

File: backend/models/stock.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    dailyPrice: {
+        type: {}
```
Comment (mdumandag) [inline backend/models/stock.js]: Same as the comment about `monthlyPrice`

File: backend/models/stock.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    stockSymbol:{
+        type:String
```
Comment (mdumandag) [inline backend/models/stock.js]: Is this `required` ?

File: backend/routes/stock.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+const stockService = require('../services/stock');
+
+router.get('/',async (req,res)=>{
+    try{
+        const response = await stockService.getAll();
+        res.status(200).json(response);
+
+
+    }catch (e) {
+        res.status(503)
```
Comment (mdumandag) [inline backend/routes/stock.js]: Will the error handling be addressed in another PR ?

File: backend/services/stock.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    response = JSON.parse(response);
+    if(response['Meta Data']){
+        return response['Time Series (5min)'];
+    }else{
+        console.log("Error in daily Data")
+    }
+
+}
+
+const getMontlyPrice = async (symbol) => {
```
Comment (mdumandag) [inline backend/services/stock.js]: typo, `getMontlyPrice` should be `getMonthlyPrice `

File: backend/swagger.json [hunk 6]
```diff
@@ -188,6 +188,120 @@
           }
         }
       }
+    },
+    "/stock": {
+      "get": {
+        "tags": [
+          "stock"
```
Comment (mdumandag) [inline backend/swagger.json]: This tag should also be added to `tags` (`tags` section start at line 10)

File: npm-debug.log [hunk 7]
```diff
@@ -0,0 +1,19 @@
+0 info it worked if it ends with ok
```
Comment (mdumandag) [inline npm-debug.log]: Leftover ?

Comment (ikamaci) [reply npm-debug.log]: I don't know what it is. So I will remove it

File: backend/models/stock.js (hunk 3 above)
Comment (ikamaci) [reply backend/models/stock.js]: Yes I will add

File: backend/routes/stock.js (hunk 4 above)
Comment (ikamaci) [reply backend/routes/stock.js]: I have tried to implement the endpoints quickly that's why I could not handle the errors properly.
Next, it will be handled

File: backend/services/stock.js (hunk 5 above)
Comment (ikamaci) [reply backend/services/stock.js]: ouch

File: backend/models/stock.js (hunk 1 above)
Comment (ikamaci) [reply backend/models/stock.js]: It's a object but kind of weird time-series data, I will changed it to `Object`
https://www.alphavantage.co/query?function=TIME_SERIES_INTRADAY&symbol=AAPL&interval=5min&apikey=FZD9JFSVI7D6HY5E
